### PR TITLE
Demo Frontend -- Backend changes

### PR DESCRIPTION
**Pull request by awaisdar001**
Setting up backed for MicroFrontend -- V1
---
### Code Review
  - [ ] @DawoudSheraz 

### Ready to merge? 

  - [x] You've run `make test` - if there are test failures, do not proceed. 
  - [ ] You have atleast one :+1
  - [ ] ~~You have run `make quality`~~ _Under Progress_

### CODE REVIEW

File: djreact/djangoapps/trips/managers.py [hunk 1]
```diff
@@ -17,6 +17,24 @@ def get_queryset(self):
         Usage:
             >>> TripSchedule.available.all()
         """
-        return super(AvailableTripsManager, self).get_queryset().filter(
+        return super(AvailableTripScheduleManager, self).get_queryset().filter(
```
Comment (DawoudSheraz) [inline djreact/djangoapps/trips/managers.py]: For a trip, will we be getting the active schedules by trip.trip_schedules.available.all()?

Comment (awaisdar001) [reply djreact/djangoapps/trips/managers.py]: You have raised a very good question, Which I would love to clarify. 

For getting available schedules you would
`trip.trip_schedule(manager='available').all()`
and for getting all the schedule

`trip.trip_schedule.all()`

Comment (DawoudSheraz) [reply djreact/djangoapps/trips/managers.py]: cool. Thanks for the explanation.

File: djreact/djangoapps/trips/admin.py [hunk 2]
```diff
@@ -30,6 +30,12 @@ class TripAdmin(admin.ModelAdmin):
     list_filter = ('starting_location',)
     search_fields = ['name', 'description', 'locations_included']
 
+    def save_model(self, request, obj, form, change):
+        import pdb
+        pdb.set_trace()
```
Comment (DawoudSheraz) [inline djreact/djangoapps/trips/admin.py]: `pdb` lines to remove

**Pull request merged**